### PR TITLE
Add glob matching

### PR DESCRIPTION
**Pull request by Hugal31**
Tracking issue: #7 

- Add glob matching with defaults patterns:
  - `Cargo.toml`
  - `Cargo.lock`
  - `src/**/*.rs`
  - `tests/**/*.rs`
  - `build.rs`
- Add `--include` option to add custom patterns.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
     ignore_duration: Duration,
     project_dir: Option<PathBuf>,
-    cargo_test_args: Vec<&'a str>
+    cargo_test_args: Vec<&'a str>,
+    patterns: Vec<&'a str>,
 }
 
 impl<'a> ConfigBuilder<'a> {
     pub fn new() -> Self {
+        let defaults_pattenrs = vec![
```
Comment (greyblake) [inline src/config.rs]: There is a little typo in `pattenrs`

File: src/lib.rs [hunk 2]
```diff
@@ -24,22 +25,33 @@ pub fn run() {
             .version("0.2.0")
             .author("Sergey Potapov <blake131313@gmail.com>")
             .about("Automatically runs tests for Rust project and notifies about the result.\nSource code: https://github.com/greyblake/cargo-testify")
+            .arg(Arg::with_name("includes")
+                 .short("i")
+                 .long("include")
+                 .takes_value(true)
+                 .help("Comma separated list of include pattern"))
```
Comment (greyblake) [inline src/lib.rs]: I would add `in addition to the predefined default patterns`

**Comment by greyblake** [comment]
@Hugal31 Thank you for your PR. I am ready to merge this after addressing few very minor notes.

**Comment by Hugal31** [comment]
Done! Thank you for the review!

**Comment by greyblake** [comment]
@Hugal31 Thank you! Sorry for the delay. 
I will publish a new version to crates.io this weekend.

**Pull request merged**